manual/project-aurora/project-aurora.nuspec

```diff
@@ -64,12 +64,10 @@
     <packageSourceUrl>https://github.com/kwilliams1987/chocolatey-packages/tree/master/manual/project-aurora</packageSourceUrl>
     <docsUrl>https://www.project-aurora.com/help.html</docsUrl>
     <mailingListUrl>https://github.com/antonpup/Aurora/issues</mailingListUrl>
+    <licenseUrl>https://github.com/antonpup/Aurora/blob/master/LICENSE</licenseUrl>
     <tags>rgb logitech corsair razer clevo cooler master steelseries wooting roccat alienware playStation drevo soundblaster asus yeelight nzxt</tags>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <iconUrl>https://raw.githubusercontent.com/antonpup/Aurora/master/Project-Aurora/Project-Aurora/Resources/aurora_logo_256x256.png</iconUrl>
-	<dependencies>
```

Comment:
Did you test this in the `chocolatey-test-environment`?

I originally needed to add the dependency so it would pass validation.

Reply:
Interesting, I did neglect to do that.  But a huge number of choco packages (probably the vast majority) don't have deps.  Take for instance, the base extension itself:
https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/extensions/chocolatey-core.extension/chocolatey-core.extension.nuspec

Maybe the validator doesn't like it if there's an empty dependency tag.  Better safe than sorry then based off of your experience with the package.  Most users won't hurt with an extra 15MB.